src/library_fs.js

```diff
@@ -1053,9 +1053,11 @@ FS.staticInit();
         mode = 0;
       }
       var node;
+      var isDirPath;
```

Comment:
Does this need to be declared up here at this level?

Reply:
Sorry I misread the diff.  Ignore this.

Reply:
Right. Well `var` doesn't respect groups so it doesn't _need_ to be declared up here, but I like to pretend that `var` is scoped like `let`...